app/models/physical_server.rb

```diff
@@ -6,6 +6,7 @@ class PhysicalServer < ApplicationRecord
   belongs_to :ext_management_system, :foreign_key => :ems_id, :class_name => "ManageIQ::Providers::PhysicalInfraManager"
 
   default_value_for :enabled, true
+  has_one :host, :foreign_key => "service_tag", :primary_key => "serial_number"
```

Comment:
Greg,  does a physical server belong to a host or does the host belong to the physical server?  I was thinking that a physical server has a host, which has 1 - many VMs.   Is our thinking backwards on this?